modules/lvms-creating-lvmcluster-using-rhacm.adoc

```diff
@@ -51,7 +51,7 @@ spec:
 ----
 <1> Contains the configuration to assign the local storage devices to the LVM volume groups.
 <2> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
-<3> Contains the LVM thin pool configuration.
+<3> Contains the configuration to create a thin pool in the LVM volume group. If you exclude this field, logical volumes are thick-provisioned.
```

Comment:
```suggestion
<3> Contains the configuration to create a thin pool in the LVM volume group. If you exclude this field, logical volumes are thick provisioned.
```

Reply:
Nit and kind of out of scope: Typically, we use the term _specifies_ instead of _contains_ to start code callouts. This is the most common practice and you can see lots of examples of this in the repo guidelines. There are other leading verbs used depending on the example and prior art, so it isn't a hard and fast rule. Personally, I try to use _specify_ whenever possible so I don't have to think about it. 

This is a minor issue, so if you want to create a follow up ticket to fix it in the docs set at a later time, that is your call. Though I do think that for new documentation and future PRs, I encourage you to adopt _specify_/_specifies_ when it makes sense. It certainly makes my life easier as a writer.

(But definitely not a blocker).